testnet/tools/nns-tools/lib/proposals.sh

```diff
@@ -270,6 +281,33 @@ generate_sns_bless_wasm_proposal_text() {
     ESCAPED_IC_REPO=$(printf '%s\n' "$IC_REPO" | sed -e 's/[]\/$*.^[]/\\&/g')
     RELATIVE_CODE_LOCATION="$(echo "$CANISTER_CODE_LOCATION" | sed "s/$ESCAPED_IC_REPO/./g")"
 
+    # If the canister has an unrelease_changelog.md file, use that to populate
+    # the "Features & Fixes" section of the proposal.
+    FEATURES_AND_FIXES="TODO Hand-craft this section."
+    PRIMARY_RELATIVE_CODE_LOCATION=$(echo "${RELATIVE_CODE_LOCATION}" | cut -d' ' -f1)
+    UNRELEASED_CHANGELOG_PATH="${PRIMARY_RELATIVE_CODE_LOCATION}/unreleased_changelog.md"
```

Comment:
ditto - this path will not work if not running from root of repository (i.e. if you're in the ns-tools directory).  See the PR in the other comment.